gammapy/background/models.py

```diff
@@ -71,3 +81,775 @@ def evaluate(self, x, y):
+        u_energy = '{0.unit:FITS}'.format(self.energy_bins)
+        u_bg = '{0.unit:FITS}'.format(self.background)
+
+        # data arrays
+        a_detx_lo = Quantity([self.detx_bins[:-1]])
+        a_detx_hi = Quantity([self.detx_bins[1:]])
+        a_dety_lo = Quantity([self.dety_bins[:-1]])
+        a_dety_hi = Quantity([self.dety_bins[1:]])
+        a_energy_lo = Quantity([self.energy_bins[:-1]])
+        a_energy_hi = Quantity([self.energy_bins[1:]])
+        a_bg = Quantity([self.background])
+
+        # table
+        table = Table()
+        table['DETX_LO'] = a_detx_lo
+        table['DETX_HI'] = a_detx_hi
+        table['DETY_LO'] = a_dety_lo
+        table['DETY_HI'] = a_dety_hi
+        table['ENERG_LO'] = a_energy_lo
+        table['ENERG_HI'] = a_energy_hi
+        table['Bgd'] = a_bg
+
+        table.meta['E_THRES'] = a_energy_lo.flatten()[0].value
+
+        # name
+        table.meta['name'] = 'BACKGROUND'
+
+        return table
+
+    def to_fits_bin_table(self):
+        """Convert cube background model to binary table fits format.
+
+        Returns
+        -------
+        tbhdu : `~astropy.io.fits.BinTableHDU`
+            table containing the bg cube
+        """
+        # build astropy table
+        table = self.to_astropy_table()
+
+        # read name and drop it from the meta information, otherwise
+        # it would be stored as a header keyword in the BinTableHDU
+        name = table.meta.popitem('name')[1]
+
+        data = table.as_array()
+
+        header = fits.Header()
+        header.update(table.meta)
+
+        tbhdu = fits.BinTableHDU(data, header, name=name)
+
+        # Copy over column meta-data
+        for colname in table.colnames:
+            tbhdu.columns[colname].unit = str(table[colname].unit)
+
+        # TODO: this method works fine but the order of keywords in the table
+        # header is not logical: for instance, list of keywords with column
+        # units (TUNITi) is appended after the list of column keywords
+        # (TTYPEi, TFORMi), instead of in between.
+        # https://github.com/gammapy/gammapy/issues/298
+
+        return tbhdu
+
+    def to_fits_image(self):
+        """Convert cube background model to image fits format.
+
+        Returns
+        -------
+        hdu : `~astropy.io.fits.ImageHDU`
+            image containing the bg cube
+        """
+        hdu = fits.ImageHDU(data=self.background.value)
+        # TODO: store units (of bg) somewhere in header??!!!!
+        # TODO: implement WCS object to be able to read the det coords -> done
+        # TODO: energy binning: store in HDU table like for SpectralCube class
+
+        # get WCS object
+        wcs = self.det_wcs
+
+        # Now, write out the WCS object as a FITS header
+        wcs_header = wcs.to_header()
+
+        # transfering header values
+        # need to copy necessary values one by one
+        # TODO: can this be simplified? just copy the header doesn't
```

Comment:
I'm still sure that copying header keywords one by one isn't needed.

Can you try some more, e.g. does this work?
```python
hdu.header.update(wcs_header)
```
Or maybe you have to convert `wcs_header` to a dict first?
http://astropy.readthedocs.org/en/latest/io/fits/api/headers.html#astropy.io.fits.Header.update

If you can't figure this out by adding
```
import IPython; IPython.embed()
```
here and then playing around for 15 minutes, or by browsing / searching the Astropy fits / wcs docs, let me know and I'll try myself.